src/main/logutils.ts

```diff
@@ -575,6 +696,32 @@ function zoneChangeStop (line: LogLine): void {
     recorder.stop(metadata);
 }
 
+/**
```

Comment:
Not sure how I feel about this. What about horribly bad runs with like 20 deaths, is that all going to appear in the context menu? Might get a bit busy. What about hunter feign deaths which causes a UNIT_DIED event to fire?

Reply:
> What about horribly bad runs with like 20 deaths, is that all going to appear in the context menu?

We will be better and die less! Just kidding, I get what you mean - especially on raid bosses where several players might die in close succession either because of mechanics of due to a wipe being called.

I _could_ make some kind of logic such that a player death would only count if there has been no other player deaths in X number of seconds, or simply ignore player deaths after a certain number.

For now, player deaths aren't exposed in the UI - I created the logic for it but ultimately decided to leave it out due to space issues - we'll talk about what to do with it after M+ is merged in, as discussed on Discord too. 

Does that sound OK?

> What about hunter feign deaths which causes a UNIT_DIED event to fire?

Good point! I can actually see here, that the very last value in the `UNIT_DIED` line indicates if the unit is "unconscious" (i.e. feigned death) so I can/will filter on that. Shouldn't have feign deaths included.

Reply:
Oh, that's awesome on the feign death thing. I had no idea that was available. That'd be great to include. 

Ultimately I think I don't care about this now I know it's not ending up in the context menu!